src/components/ApplicationGroup/Details/TriggerView/BulkCDTrigger.tsx

```diff
@@ -426,16 +428,16 @@ export default function BulkCDTrigger({
             )
         }
 
+        if (app.triggerBlockedInfo?.blockedBy === TriggerBlockType.MANDATORY_TAG) {
```

Comment:
Please set warning and ensure material calls are not going in case we have this error